heritago/heritago/urls.py

```diff
@@ -15,8 +15,28 @@
 """
 from django.conf.urls import url, include
 from django.contrib import admin
+from heritages import views
+from django.contrib.auth.views import password_reset
+
+
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r"api/v1/heritages/", include("heritages.urls"))
+    url(r"api/v1/heritages/", include("heritages.urls")),
+
+# user auth urls
+
+   # url(r'^$', views.diary, name='home'),
+    url(r'^login/$', views.login, name='login'),
+    url(r'^auth/$', views.auth_view, name='auth_view'),
+    url(r'^logout/$', views.logout, name='logout'),
+    url(r'^loggedin/$', views.loggedin, name='loggedin'),
```

Comment:
such routers seem to break RESTful architecture, I think. Since our backend side is not aware of client-side usage, routers like register_success, loggedin should not be defined here.